src/commands/create/server/ipblocks/createserver_ipblock.go

```diff
@@ -0,0 +1,55 @@
+package ipblocks
+
+import (
+	"github.com/spf13/cobra"
+	"phoenixnap.com/pnapctl/common/client/bmcapi"
+	"phoenixnap.com/pnapctl/common/models/bmcapimodels/servermodels"
+	"phoenixnap.com/pnapctl/common/printer"
+	"phoenixnap.com/pnapctl/common/utils"
+)
+
+// Filename is the filename from which to retrieve a complex object
+var Filename string
+
+var commandName = "create server-ip-block"
```

Comment:
Since ip blocks are created at network-configuration level (just like private networks), I decided to use the same standard we adopted for private networks. If in the future we face this situation you are talking about then we can introduce a new command along the lines of `pnapctl create server-private-network-ip-block  <SERVER_ID> <PRIVATE_NETWORK_ID>`

Reply:
My suggestion was accepted in an offline conversation. Resolving.